Strain/StrainVisualization.h

```diff
@@ -0,0 +1,75 @@
+#ifndef STRAIN_VISUALIZATION_H
+#define STRAIN_VISUALIZATION_H
+
+#include <SFML/Graphics.hpp>
+
+#include "StrainGauge.h"
+
+
+/**
+ * \brief Class of visualising data from the strain gauge
+ */
+class StrainVisualization : public sf::Drawable, public StrainGauge
+{
+	/**
+	 * \brief Array of data from the sensor
+	 */
+	std::array<int, 6> _data;
+
+	/**
+	 * \brief Limit of the X axis on the chart
+	 */
+	const int _limitX;
+
+	/**
+	* \brief Limit of the Y axis on the chart
+	*/
+	const int _limitY;
+
+	/**
+	* \brief Vectors of the data on 6 axises
+	*/
+	std::vector<int> _y1, _y2, _y3, _m1, _m2, _m3;
+
+	/**
+	 * \brief Data for correcting
+	 */
+	std::array<int, 6> _calibrate;
+
+	/**
+	* \brief Arrays of vertexes to draw on the screen
+	*/
+	sf::VertexArray _forceX, _forceY, _forceZ, _torqueX, _torqueY, _torqueZ;
+
+	/**
+	* \brief Arrays of vertexes to draw coordinate system on the screen
+	*/
+	sf::VertexArray _coordSystem;
+
+	/**
+	 * \brief Function to draw in the screen
+	 * \param target Hui znaet
```

Comment:
**(must be changed)**: change this comments, add `param[in]/[out]`.

Reply:
Done